test/plugins/prefixIds.test.js

```diff
@@ -22,3 +22,51 @@ test('should extract prefix from path basename', () => {
     }).data
   ).toEqual(`<svg id="input_svg__my-id"/>`);
 });
+
+test('should replace references in attributes', () => {
```

Comment:
This test module is used only for testing filename. All tests are in `test/plugins/prefixIds.*.svg`

Reply:
Oh, I didn't found the right place for plugin tests. Writing tests in *.svg files is a bit... weird :)  So I thought that prefixIds plugin has only this single test and added some more. Now I removed my redundant tests and added one for testing `style="..."` replacement as `prefixIds.11.svg`.